bin/PerlDDS/Run_Test.pm

```diff
@@ -464,7 +464,132 @@ sub wait_kill {
 
   my $process = $self->{processes}->{process}->{$name}->{process};
 
-  return PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+  my $result = PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+
+  # If opts has indicated that user wants to get core dump/signal information
+  # then find the core dump file if any and print its stack trace.
+  if (defined $opts && defined $opts->{dump_ref} && ${$opts->{dump_ref}}) {
+      # Get the core file pattern
```

Comment:
Is the indent right for this block (and children)? It looks like it's 2 spaces elsewhere in this file.

Reply:
Yeah, the indentation is not right. I'll fix it.